src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpGatewayOrganisationChecksController.cs

```diff
@@ -53,7 +54,7 @@ public async Task<IActionResult> EvaluateLegalNamePage(LegalNamePageViewModel vi
             if (validationResponse.Errors != null && validationResponse.Errors.Any())
             {
                 viewModel.ErrorMessages = validationResponse.Errors;
-                return View("~/Views/Roatp/Apply/Gateway/pages/LegalName.cshtml", viewModel);
+                return View($"{GatewayViewsLocation}LegalName.cshtml", viewModel);
```

Comment:
As mention above, splitting out the training slash makes it more readable. So it would become:
`return View($"{GatewayViewsLocation}/LegalName.cshtml", viewModel);`

Reply:
agree